### PR TITLE
comp added : typography list nav modal , bug fixes

### PR DESCRIPTION
**Pull request by manojsarna**
Added Typogrpahy

Added List

Added Navigation

Added Modal

Some minor changes

### CODE REVIEW

**Review by j24m — APPROVED** [review]
Perfection. Good to merge .!!💯

**Comment by ashishkatiyar2404** [comment]
You can Increase the transition timing on hovering get a start and GitHub button just for smoothness otherwise it still looks awesome👌

you can Increase the margin between the buttons(read and bookmark) and text on Text Only Cards

**Review by m26unkwn — APPROVED** [review]
Dark Mode is super Cool.
Things that are too cool.

1. Navigation
2. List
3.  Hard-coding the heading elements

File: lib/Components/Typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+.fw-700 {
+  font-weight: 700;
+}
+.fw-800 {
+  font-weight: 800;
+}
+.fw-900 {
+  font-weight: 900;
+}
+
```
Comment (m26unkwn) [inline lib/Components/Typography/typography.css:47]: This is something new.

**Review by m26unkwn — APPROVED** [review]

**Review by ashishkatiyar2404 — COMMENTED** [review]
All good Manoj your code looks good to me and is easy to read.
UI also looks awesome.

File: lib/doc/doc.html [hunk 2]
```diff
@@ -2070,8 +2560,10 @@ <h4>Made with ❤️ by Manoj Sarna</h4>
     </footer>
```
Comment (mayuresh-tech) [inline lib/doc/doc.html:2560]: Clean code
Just a suggestion to add footer with social media links

**Comment by ritikaawasthy** [comment]
Your code looks clean and dark theme is really nice

**Review by Cl0udCode — COMMENTED** [review]
### nice way to implement reusable code

**Pull request merged**